### PR TITLE
Add documentation of the new GC options for JVM version > 9

### PR DESCRIPTION
**Pull request by guerremdq**
All the documented options are deprecated in the latest version of the JVM.

### CODE REVIEW

File: presto-docs/src/main/sphinx/admin/tuning.rst [hunk 1]
```diff
@@ -30,3 +30,8 @@ The following can be helpful for diagnosing garbage collection (GC) issues:
     -XX:+PrintAdaptiveSizePolicy
     -XX:+PrintSafepointStatistics
     -XX:PrintSafepointStatisticsCount=1
+
+If Java JMV Version > 9:
```
Comment (martint) [inline presto-docs/src/main/sphinx/admin/tuning.rst]: Typo: JMV

Comment (martint) [reply presto-docs/src/main/sphinx/admin/tuning.rst]: Also, it should be ">= 9". Maybe rephrase as: "If running on Java 9 or later:"

Comment (findepi) [reply presto-docs/src/main/sphinx/admin/tuning.rst]: @martint since `<9` is kind of passé, do we need this?

**Comment by findepi** [comment]
Merged as ebf1ccaae8fe1e9ad7f40430082c20824131201e, thanks!
